### PR TITLE
New FAQ page

### PR DESCRIPTION
**Pull request by bshyong**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3504

screen shot 2017-08-23 at 12 20 55 pm

screen shot 2017-08-23 at 12 34 23 pm

### CODE REVIEW

File: src/js/common/components/authentication/LoginPrompt.jsx [hunk 1]
```diff
@@ -45,10 +45,14 @@ class LoginPrompt extends React.Component {
               <li>View Post-9/11 GI Bill enrollment information</li>
             </ul>
             <p>
-              <button className="usa-button-primary va-button-primary usa-button-big" onClick={this.handleLogin}><strong>Sign In</strong></button>
-              <button className="va-button-secondary usa-button-big" onClick={this.handleSignup}><strong>Create Account</strong></button>
+              <button className="usa-button-primary va-button-primary usa-button-big" onClick={this.handleLogin}><strong>Log in</strong></button>
+              <button className="va-button-secondary usa-button-big" onClick={this.handleSignup}><strong>Sign up</strong></button>
             </p>
             <p>Even without an account, you can apply for VA benefits.</p>
+            <h5>Having trouble signing in?</h5>
```
Comment (U-DON) [inline src/js/common/components/authentication/LoginPrompt.jsx]: I wonder why we're changing the language from "Sign In" to "Log In" but using "signing in" here.

**Review by U-DON — APPROVED** [review]
LGTM.

File: src/js/common/components/authentication/LoginPrompt.jsx (hunk 1 above)
Comment (U-DON) [inline src/js/common/components/authentication/LoginPrompt.jsx]: @melwoodard: ^

**Review by melwoodard — CHANGES_REQUESTED** [review]
A couple things:

- All phone numbers in the "What if I can't verify my identity online?" and "Need help?" accordions should be `tel` links
- In the "How do I verify" accordion, the two links to id.me aren't loading. @amybehrens do you know why?

File: src/js/common/components/authentication/LoginPrompt.jsx (hunk 1 above)
Comment (melwoodard) [inline src/js/common/components/authentication/LoginPrompt.jsx]: Let's revert this to "sign in"

File: src/js/common/schemaform/SaveInProgressErrorPage.jsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ class SaveInProgressErrorPage extends React.Component {
                   onLogin={this.reloadForm}
                   isLoggedIn={this.props.isLoggedIn}
                   loginUrl={this.props.loginUrl}
-                  onUpdateLoginUrl={this.props.updateLogInUrl}>Sign in</SignInLink>
+                  onUpdateLoginUrl={this.props.updateLogInUrl}>Log in</SignInLink>
```
Comment (melwoodard) [inline src/js/common/schemaform/SaveInProgressErrorPage.jsx]: Let's revert this to "sign in"

File: src/js/login/components/SearchHelpSignIn.jsx [hunk 3]
```diff
@@ -28,7 +28,7 @@ class SearchHelpSignIn extends React.Component {
         onUserLogout={this.props.onUserLogout}/>);
     } else {
       content = (<div>
-        <a href="#" onClick={this.props.onUserLogin}>Sign In</a><span className="signin-spacer">|</span><a href="#" onClick={this.props.onUserSignup}>Register</a>
+        <a href="#" onClick={this.props.onUserLogin}>Log In</a><span className="signin-spacer">|</span><a href="#" onClick={this.props.onUserSignup}>Sign up</a>
```
Comment (melwoodard) [inline src/js/login/components/SearchHelpSignIn.jsx]: Let's revert this to "sign in"

File: test/common/schemaform/SaveInProgressErrorPage.unit.spec.jsx [hunk 4]
```diff
@@ -52,7 +52,7 @@ describe('<SaveInProgressErrorPage>', () => {
 
     expect(findDOM.querySelector('.usa-alert').textContent).to.contain('You have been signed out.');
     expect(findDOM.querySelector('.usa-button-outline').textContent).to.contain('Back');
-    expect(findDOM.querySelector('.usa-button-primary').textContent).to.contain('Sign in');
+    expect(findDOM.querySelector('.usa-button-primary').textContent).to.contain('Log in');
```
Comment (melwoodard) [inline test/common/schemaform/SaveInProgressErrorPage.unit.spec.jsx]: Let's revert this to "sign in"

**Review by melwoodard — CHANGES_REQUESTED** [review]

**Comment by bshyong** [comment]
Changes made! The "how do I verify" links belong to ID.me but they are not loading for me either. This is the URL that is in the FAQ page:

https://vetsgov.id.me/hc/en-us/articles/229334788-How-do-I-complete-the-LOA3-identity-proofing-process-

**Comment by melwoodard** [comment]
Thanks Ben! Turns out id.me moved their FAQ. This should be the correct page: https://support.id.me/hc/en-us/articles/229334788-How-do-I-complete-the-LOA3-identity-proofing-process-

**Comment by bshyong** [comment]
Links are updated; if all looks good we can get this page launched today

**Review by melwoodard — CHANGES_REQUESTED** [review]
One more thing! I overlooked the "need help" box - can you make the phone numbers tel links there too? Other than that looks good!

**Review by melwoodard — APPROVED** [review]
Thanks, looks great!

**Pull request merged**